core/templates/pages/learner-dashboard-page/new-home-tab.component.html

```diff
@@ -0,0 +1,148 @@
+<div class="new-home-tab">
```

Comment:
Appending "new-*" here seems a bit odd. Will this be the learner dashboard page home tab going forward? If so, we should just call it home tab.

Reply:
Please update the filename here and elsewhere too.

Reply:
Hi, @kevintab95 
I have created a file named **new-home-tab** and this file is having redesigned the home-tab. I have created a new file so that I can manage the old and new home tab. Once the project gets complete I will remove the old-home-tab file and keep the redesigned home tab, also I will remove the word **new**
Thanks!